packages/grid/_modules_/grid/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -36,6 +37,10 @@ let cachedSupportsTouchActionNone = false;
 // Over 80% of the iOS phones are compatible
 // in August 2020.
 function doesSupportTouchActionNone(): boolean {
+  // The document is not defined in the server-side
+  if (typeof document === 'undefined') {
```

Comment:
I wonder if we will need an abstraction from the utils package to determine if we are on the server or not. Sometimes poeple add global and break our heuristics. e.g. https://github.com/mui-org/material-ui/blob/0230df94ca877fdd63a42c3e276f75c3a4316eea/packages/material-ui-utils/src/useEnhancedEffect.js#L3. Anyway, not important.

Reply:
I'm checking the document because it's the most obvious ("document is not defined" error), but all code examples I found in StackOverflow to fix this problem looks at the window. I'll leave as it is.